modules/updating-eus-to-eus-upgrade-cli.adoc

```diff
@@ -0,0 +1,144 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing-eus-eus-upgrade.adoc
+
+:_content-type: PROCEDURE
+[id="updating-eus-to-eus-upgrade-cli_{context}"]
+= EUS-to-EUS update using the CLI
+
+.Prerequisites
+
+* Update OpenShift CLI `oc` to the target version before each update.
+
+[IMPORTANT]
+====
+It is highly discouraged to skip this prerequisite. If the OpenShift CLI `oc` is not updated to the target version before your update, unexpected issues may occur.
+====
+
+.Procedure
+
+. Using the Administrator perspective on the web console, update any Operator Lifecycle Manager (OLM) Operators to the versions that are compatible with the version you are updating to. You can find more information on how to perform this action on the link:https://docs.openshift.com/container-platform/4.12/operators/admin/olm-upgrading-operators.adoc[Updating installed Operators] page.
+
+. Verify that all machine config pools display a status of `UPDATED` and that no machine config pool displays a status of `UPDATING`.
```

Comment:
Thank you! Does the step on web-UI currently provide those details? I'm under the impression that the user is checking to make sure the pools are updated, not whether they are paused or unpaused.

For web-ui, we have the user check the status through "click *Compute* -> *MachineConfigPools* and review the contents of the *Update status* column."

Are you suggesting that we need a step that reveals more info than the ask to run`$ oc get mcp` ?

Reply:
well, since that "EUS-to-EUS update" section is relevant to both webui as well as cli, it's up to you if you want to remind the admin about verifying the pools unpaused before checking if they are updated in "EUS-to-EUS update using the CLI"